examples/gallery/embellishments/pygmt_logo.py

```diff
@@ -0,0 +1,362 @@
+            fig.plot(x=x_coords, y=y_coords, **args_m)
+        # middle corner
+        fig.plot(x=0.9, y=0.9, style="d0.3c", fill=color_red, perspective=True)
+
+        # .............................................................................
+        # letter T
+        # .............................................................................
+        # red curved horizontal line
+        fig.plot(
+            x=0, y=0, style="w4.6c/240/-60+i3.7c", fill=color_red, perspective=True
+        )
+        # vertical endings of curved horizontal line
+        args_vert = {"y": [-1.5, -2.5], "pen": f"9p,{color_bg}", "perspective": True}
+        fig.plot(x=[-1.05, -1.05], **args_vert)
+        fig.plot(x=[1.05, 1.05], **args_vert)
+        # arrow head as inverse triangle with pen for space to blue circle / hexagon
+        fig.plot(
+            x=0,
+            y=-3.55,
+            style="i1.1c",
+            fill=color_red,
+            pen=f"3p,{color_bg}",
+            perspective=True,
+        )
+        # arrow tail
+        fig.plot(x=[0, 0], y=[-2, -3.57], pen=f"12p,{color_red}", perspective=True)
+
+        # margin around shape for black_white in dark_mode - Needed ???
+        if black_white and dark_mode:
+            fig.plot(
+                x=0,
+                y=0,
+                style=f"{symbol}{diameter + diameter_add}c",
+                pen=f"1p,{color_dark}",
+                no_clip=True,
+                perspective=True,
+            )
+
+        # .............................................................................
+        # Save
+        # .............................................................................
+        # fig.show()
+        fig_name_rot = fig_name_logo = "pygmt_logo_rot"
+        fig.savefig(fname=f"{fig_name_rot}.eps")
+        # print(fig_name_rot)
+
+        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        # Replot and add wordmark "PyGMT"
+        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        if wordmark is True:
+            fig = pygmt.Figure()
+            pygmt.config(MAP_FRAME_PEN=no_line)
+            fig.basemap(region=region, projection=proj_wm, frame=no_fill)
+
+            fig.image(imagefile=f"{fig_name_rot}.eps", position=pos_wm)
+
+            # Try GMT color setting to avoid re / overplotting
+            text_wm = f"@;{color_py};Py@;;@;{color_gmt};GMT@;;"
+            fig.text(text=text_wm, font="45p,AvantGarde-Book", **args_text_wm)
+
+            # .........................................................................
+            # Save
+            # .........................................................................
+            fig_name_rot_wm = fig_name_logo = f"{fig_name_rot}_wordmark_{orientation}"
+            fig.savefig(fname=f"{fig_name_rot_wm}.eps")
+            # print(fig_name_rot_wm)
+            Path.unlink(f"{fig_name_rot}.eps")
+
```

Comment:
No need to create a new Figure instance, just add "PyGMT" to the right of the visual logo.
```suggestion
        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        # Add wordmark "PyGMT"
        # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        if wordmark is True:
            text_wm = f"@;{color_py};Py@;;@;{color_gmt};GMT@;;"
            fig.text(text=text_wm, no_clip=True, **args_text_wm)
        fig_name_logo = "pygmt_logo"
        fig.savefig(fname=f"{fig_name_logo}.eps")
```

Reply:
Thanks 🙂!
Since now the box around the logo is added after creating the entire logo (visual and wordmark) via the `box` parameter of `Figure.image`, it works to add the wordmark outside of the plotting region. Initially this did not look good because the box was based on the plotting region by fill it with a color.